### PR TITLE
Add contributing conventions

### LINKED ISSUE #1: Please comment here!
Please comment on this particular issue. Anything will do.

### PR DESCRIPTION
**Pull request by githubteacher**
@riosu You thoughts?
- [ ] task 1
- [ ] task 2

Closes #1

### CODE REVIEW

**Comment by dice** [comment]
@githubteacher   I could not think of any good rules, but I did add this photo.
contemporary-home-decor

**Comment by iwashi** [comment]
:sushi:

**Comment by ewanleaver** [comment]
:sheep: :sheep: :sheep: :sheep: :sheep:

**Comment by mmasaki** [comment]
:+1:

**Comment by alanmshelly** [comment]
all comments should contain a :hamster:

**Comment by riosu** [comment]
:dancer:

**Comment by riosu** [comment]
:dancer:

**Comment by mmasaki** [comment]
:shipit::shipit::shipit::shipit:

**Comment by ewanleaver** [comment]
:ship:

**Comment by ikeike443** [comment]
寿司ゆき:yes

**Comment by mmasaki** [comment]
#1

**Comment by githubteacher** [comment]
Context: #3

**Comment by ikeike443** [comment]
Smith & Franklin - Academic Publishing Corporation

**Comment by dice** [comment]
@githubteacher here's the help page!

https://help.github.com/articles/closing-issues-via-commit-messages/

:bow:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contribution flow
+
+## Coding Conventions
```
Comment (githubteacher) [inline CONTRIBUTING.md:3]: @dice What are some of your pet peeves?

Comment (ikeike443) [inline CONTRIBUTING.md:3]: objection!
4Gamer.net ― 【速報】「逆転裁判5」制作決定＆iOS版「逆転裁判 123HD ...

Comment (dice) [inline CONTRIBUTING.md:3]: @githubteacher I do not like it when code comments are written in ALL CAPS. :blush:

**Comment by githubteacher** [comment]
:ship:

**Pull request merged**